aiconfig-docs/docs/gradio-workbook.md

```diff
@@ -0,0 +1,144 @@
+
+## Quickstart
+
+### 1. Create a new space with Gradio Workbooks
+
+- **Start with a template**. Duplicate the [Gradio Workbook Quickstart space](https://huggingface.co/spaces/lastmileai/gradio-workbook-template)
+- **Create from scratch**. Create a [new Gradio SDK space](https://huggingface.co/new-space) and add these files to your space repo. To add files to your space repo, you can do so through the [Web UI](https://huggingface.co/docs/hub/repositories-getting-started#adding-files-to-a-repository-web-ui) or [terminal](https://huggingface.co/docs/hub/repositories-getting-started#terminal).
+  - [app.py](https://huggingface.co/spaces/lastmileai/gradio-workbook-template/blob/main/app.py)
+  - [requirements.txt](https://huggingface.co/spaces/lastmileai/gradio-workbook-template/blob/main/requirements.txt)
+
+### 2. Design your space
+
+- Add models and customize. Edits/changes won’t save automatically.
+
+### 3. Lock in your edits
+
+- Click **Download**, which downloads the `aiconfig` file. This captures the current configuration of your space.
+- Rename the downloaded file to `my_app.aiconfig.json`.
+- Replace `my_app.aiconfig.json` in the space repo with the downloaded one.
+
+Anyone who visits your space will see the state represented by `my_app.aiconfig.json`.
+
+## Examples
+
+- [Music Playground](https://huggingface.co/spaces/lastmileai/music_generator). Make your demo more captivating with different modalities.
+- [AI Story Generator](https://huggingface.co/spaces/lastmileai/ai_story_generator). Easy to chain models for a workflow like story building.
+- [Stable Diffusion XL Playground](https://huggingface.co/spaces/lastmileai/sdxl_playground). Create interactive prompt templates for your model.
+
+## Features
+
+### Gradio Workbook Structure
+
+Gradio Workbooks are inspired by Jupyter notebooks. Each workbook is made up of cells. The cell is associated with a model and running the cell executes your prompt on that model.
+
+<div align="center">
+  <picture>
+    <img alt="workbook" src="https://github.com/lastmile-ai/aiconfig/assets/81494782/85c9953f-0cf6-4d0c-84bf-5fffa4e5bcf5" width="800"/>
+  </picture>
+</div>
+
+### Parameters
+
+Parameters are variables that you can define and pass into any cell within your workbook. You can set both **global parameters** and **local parameters**. For example, you can define parameters such as language or product in a cell like this with the handlebars syntax as shown below: `"Write a poem in {{language}} about a salesman selling {{product}}."`
+
+#### Global Parameters
+
+You can set global parameters for your entire workbook. Click on **Global Parameters {}** on the top of your workbook and set your global parameters that can be used in any cell of the workbook.
+
```

Comment:
Let's also show screenshot of this. This is kind of small but example here:

<img width="840" alt="Screenshot 2024-02-01 at 10 27 52" src="https://github.com/lastmile-ai/aiconfig/assets/151060367/6a497f45-397d-471b-8b21-6cbad5a7a9aa">

Reply:
Actually nm, I feel like your example is pretty good since it relates to the screenshot. In that case let's also not use `"Write a poem in {{language}} about a salesman selling {{product}}."` because that doesn't have anything to do with the original example screenshot with the arrows etc